implementation/core/src/main/java/io/smallrye/faulttolerance/core/fallback/FallbackEvents.java

```diff
@@ -0,0 +1,21 @@
+package io.smallrye.faulttolerance.core.fallback;
+
+import io.smallrye.faulttolerance.core.InvocationContextEvent;
+
+public class FallbackEvents {
+    public static class Defined implements InvocationContextEvent {
```

Comment:
why not enum values? Sth like:
public enum FallbackEvents {
   DEFINED,
   APPLIED
}